### ISSUE TITLE
SVG export via Export API stopped working

### ISSUE THREAD

**Issue by jaromirdalecky** [amcharts/amcharts4#3003]
**Bug description**

When I attempt to export a chart as SVG using the export API without passing options

```
chart.exporting.getSVG().then(function(data) {
    console.log(data);
});
```

I get an error `TypeError: Cannot read property 'scale' of undefined`

CodePen: https://codepen.io/czjata/pen/OJXyXoQ

This seems to be related to the recent addition of SVG export options.

**Environment**

amCharts 4.10.7

**Comment by martynasma** [comment]
Thanks for the heads up!

This issue was fixed in dev version, and will be released soon.

We'll notify you here when that happens.

**Comment by martynasma** [comment]
Fixed in [4.10.8](https://github.com/amcharts/amcharts4/releases/tag/4.10.8).

## [4.10.8] - 2020-10-19

### Added
- New `Tooltip` property `ignoreBounds` (default `false`). If set to `true`, will not try to contain tooltips within the boundaries of the chart.

### Fixed
- Export: Invoking export's `getSVG()` method without passing in options as a second parameter was resulting in error.
- Series' actual value was not being properly updated in all cases when used with `gropuDataItem` adapter.
- Series with with extremely small value range could result in an SO.
- Resetting data to an empty data set on an `XYSeries` witth bullets and a data grouping enabled, could result in bullets being left on the screen.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by jaromirdalecky** [comment]
@martynasma Thanks but it is fixed only partially in my opinion unless it is deliberate decision.

Previously I could export the SVG just with:
chart.exporting.getSVG().then(function(data) {
    console.log(data);
});
```

With the fix 4.10.8 applied I still need to pass the type explicitly:
```
chart.exporting.getSVG("svg").then(function(data) {
    console.log(data);
});
```

I've updated my original CodePen.

**Comment by martynasma** [comment]
Well, technically type parameter is required. But we'll make it so it does not fail if it's not in the next version.

I'll notify you here when new version is out (this or next week).

**Comment by martynasma** [comment]
Fixed in [4.10.9](https://github.com/amcharts/amcharts4/releases/tag/4.10.9).

## [4.10.9] - 2020-10-23

### Changed
- Export: API functions like `getCSV`, `getSVG`, `getJSON`, `getExcel` will not error out if no opeionts or format identifier is passed in.

### Fixed
- Better invalidation of charts with `am4core.options.queue = true` enabled.
- A slice in `PieSeries` remained enlarged (as hovered) after subsequent taps on it on a touch device.
- In some rare cases, on mobile devices a JS error could be invoked when disposing series/axes.
- Vertical `ValueAxis` was not auto-zooming in some cases.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
